### PR TITLE
Improved footer on tablet view

### PR DESCRIPTION
**Pull request by Goodiec**
# Issue Description
This a fix for #1338 
Please include a summary of the issue.
The content columns of the footer are not properly arranged on screen sizes between 767px and 481px
Fixes # (issue)
#1338 
## Changes Made
Made changes to the ocamlorg.css and added a class to a div in the footer area in the index.html file that I used in adding padding to the top of the footer.
Please describe the changes that you made.
Change the footer CSS to display two columns per row on the mentioned screen sizes, increased the font-size of the h1 elements of the footer, and also added padding to the top of the footer so it looks better and uniforms with the padding at the bottom of the footer.
* **Please check if the PR fulfills these requirements**

- [x] PR is descriptively titled and links the original issue above
- [x] Before/after screenshots (if this is a layout change)
- [x] Details of which platforms the change was tested on (if this is a browser-specific change)
- [x] Context for what motivated the change (if this is a change to some content)

### CODE REVIEW

**Comment by gs0510** [comment]
Hi @Goodiec! Thanks for your PR! Can you add a before and after screenshot in the PR.

Also, your PR has merge conflicts which means that you need to rebase it against master.  :)

**Comment by Goodiec** [comment]
> Hi @Goodiec! Thanks for your PR! Can you add a before and after screenshot in the PR.
> 
> Also, your PR has merge conflicts which means that you need to rebase it against master. :)

Here's the before and after screenshots 

before
after

**Comment by Goodiec** [comment]
> > Hi @Goodiec! Thanks for your PR! Can you add a before and after screenshot in the PR.
> > Also, your PR has merge conflicts which means that you need to rebase it against master. :)
> 
> Here's the before and after screenshots
> 
> before
> after

@gs0510 I have rebased and resolved the conflict, kindly check. Thank you

File: template/footer.mpp [hunk 1]
```diff
@@ -53,4 +53,4 @@
     </ul>
   </div>
 </div>
-c!}}
```
Comment (gs0510) [inline template/footer.mpp:56]: Can you remove this file from the PR since nothing has been changed here? :)

Comment (Goodiec) [reply template/footer.mpp:56]: I removed the trailing white space from that file that was causing the conflict.

**Pull request closed without merging**